images/build/installDotNetCore.sh

```diff
@@ -17,14 +17,12 @@ debianFlavor="$DEBIAN_FLAVOR"
 
 fileName="dotnet.tar.gz"
 
-# TODO: PR2 configure this to account for the different debian flavors once the Version metadata has
```

Comment:
No actual changes were made in this file right?

Reply:
@cormacpayne  correct, I just wanted to make sure that I went through and double checked that the logic in this method was going to be correct following the change to the storage account